docs/overview.md

```
@@ -0,0 +1,145 @@
+  historical data spanning over 80 years is used to 
+  derive return periods, placing current forecasts 
+  in a meaningful historical context. These streamflow 
+  forecasts are accessible via web mapping services and APIs, 
+  facilitating the development of customized applications 
+  tailored to address local water management challenges 
+  such as flooding, drought, and water security.
+
+### Map
+  * The GEOGloWS ECMWF Streamflow Model can be easily explored 
+  using a mapping service developed in collaboration with 
+  Esri (see: https://hydroviewer.geoglows.org/). The web map 
```

Markdownlint

Bare URL used

(MD034, no-bare-urls)